github/WorkflowRun.py

```diff
@@ -92,6 +95,14 @@ def conclusion(self):
         self._completeIfNotSet(self._conclusion)
         return self._conclusion.value
 
+    @property
+    def workflow_id(self):
```

Comment:
This is unrelated to the Artifacts API, I'd prefer it was split out to its own tiny PR.

Reply:
Good point. Done: #1737